### DISCUSSION TITLE
Corosync Pacemaker Integration

### DISCUSSION THREAD

**Discussion by nsime298** [cockpit-project/cockpit#20142] [Ideas]
Hello,
We've built our own in-house storage array using the open source clustering services corosync and pacemaker. We're using cockpit as the frontend GUI but would love to see the clustering information and management integrated into the UI. It seems like a great match as cockpit seems geared towards a NAS/SAN feature set.

**Comment by martinpitt** [comment]
For anyone who wants to pick this up: Please have a look at https://cockpit-project.org/external/wiki/Contributing.html#further-reading . This explains the basics of how to write a cockpit page and how to easily set up your own project for it.
